client/src/utils/parser/robotCodeToSsotParsing/robotCodeToSsotParsing.js

```diff
@@ -0,0 +1,369 @@
+
+/**
+ * @category Client
+ * @module
+ */
+
+const { default: customNotification } = require("../../notificationUtils");
+
+const FOURSPACE = '    ';
+
+
+/**
+ * 
+ * @returns "uniqueId" which is just an increment from the counter in the local storage 
+ */
+const getUniqueId = () => {
+    const newId = JSON.parse(sessionStorage.getItem('idCounter')) + 1;
+    sessionStorage.setItem('idCounter', newId);
+    return newId;
+}
+
+/**
+ * #todo
+ * @returns 
+ */
+const getActivityId = () => `Activity_0ay${getUniqueId()}`
+const getEventId = () => `Event_0ay${getUniqueId()}`
+
+
+/**
+ * @description Splits the robot code into an array and deletes all empty lines 
+ * @param {String} robotCode Code from the Code editor
```

Comment:
Please check for capitalisation

Reply:
now it's that way: Okay?

```suggestion
 * @param {String} robotCode Code from the code-editor
```

Reply:
`robotCode Code from the code editor` with an empty space

Reply:
Fixed. Thanks for the suggestion